Makefile

```diff
@@ -11,20 +11,18 @@ export BASE_BRANCH OCM_BASE_BRANCH
 
 ifneq (,$(DAPPER_HOST_ARCH))
 
-# Running in Dapper
-
-include Makefile.inc
-
 ifneq (,$(filter ovn,$(_using)))
-CLUSTER_SETTINGS_FLAG = --settings $(DAPPER_SOURCE)/.shipyard.e2e.ovn.yml
+SETTINGS ?= $(DAPPER_SOURCE)/.shipyard.e2e.ovn.yml
 else
-CLUSTER_SETTINGS_FLAG = --settings $(DAPPER_SOURCE)/.shipyard.e2e.yml
+SETTINGS ?= $(DAPPER_SOURCE)/.shipyard.e2e.yml
 endif
 
-override CLUSTERS_ARGS += $(CLUSTER_SETTINGS_FLAG)
-override DEPLOY_ARGS += $(CLUSTER_SETTINGS_FLAG)
 override E2E_ARGS += --nolazy_deploy cluster1
 
+# Running in Dapper
+
```

Comment:
This should introduce the whole “running in Dapper” section, *i.e.* lines 14 and following.

Reply:
Done